extensions/wikia/StaffLog/StaffLog.events.php

```diff
@@ -1,6 +1,6 @@
 <?php
 class StaffLogger {
-	static public function log($type, $action, $userId, $userName, $userdstId, $userNamedst, $comment = "") {
+	static public function log(string $type, string $action, int $userId, string $userName, int $userdstId, string $userNamedst, $comment = "") {
```

Comment:
👍

Reply:
MySQL 5.7 stopped casting strings to int. Let's move it to PHP then :)